models/classes/runner/rubric/QtiRunnerRubric.php

```diff
@@ -51,6 +51,9 @@ public function getRubrics(RunnerServiceContext $context, $itemRef = null)
         if(!is_null($itemRef)){
             try {
                 $routeItem = $session->getRoute()->getRouteItemsByAssessmentItemRef($itemRef);
+                if ($routeItem) {
```

Comment:
`$routeItem` is always instance of `RouteItemCollection` 
https://github.com/oat-sa/qti-sdk/blob/legacy/qtism/runtime/tests/Route.php#L1007